.drone.yml

```diff
@@ -756,107 +756,107 @@ volumes:
+  environment:
+    WORKSPACE_DIR: /tmp/push-build-darwin-amd64
+- name: Check out code
+  commands:
+  - set -u
+  - mkdir -p $WORKSPACE_DIR/go/src/github.com/gravitational/teleport
+  - cd $WORKSPACE_DIR/go/src/github.com/gravitational/teleport
+  - git clone https://github.com/gravitational/${DRONE_REPO_NAME}.git .
+  - git checkout ${DRONE_TAG:-$DRONE_COMMIT}
+  - mkdir -m 0700 $WORKSPACE_DIR/.ssh && echo "$GITHUB_PRIVATE_KEY" > $WORKSPACE_DIR/.ssh/id_rsa
+    && chmod 600 $WORKSPACE_DIR/.ssh/id_rsa
+  - ssh-keyscan -H github.com > $WORKSPACE_DIR/.ssh/known_hosts 2>/dev/null
+  - chmod 600 $WORKSPACE_DIR/.ssh/known_hosts
+  - GIT_SSH_COMMAND='ssh -i $WORKSPACE_DIR/.ssh/id_rsa -o UserKnownHostsFile=$WORKSPACE_DIR/.ssh/known_hosts
+    -F /dev/null' git submodule update --init e
+  - GIT_SSH_COMMAND='ssh -i $WORKSPACE_DIR/.ssh/id_rsa -o UserKnownHostsFile=$WORKSPACE_DIR/.ssh/known_hosts
+    -F /dev/null' git submodule update --init --recursive webassets || true
+  - rm -rf $WORKSPACE_DIR/.ssh
+  - mkdir -p $WORKSPACE_DIR/go/cache
+  environment:
+    GITHUB_PRIVATE_KEY:
+      from_secret: GITHUB_PRIVATE_KEY
+    WORKSPACE_DIR: /tmp/push-build-darwin-amd64
+- name: Build Mac artifacts
+  commands:
+  - set -u
+  - cd $WORKSPACE_DIR/go/src/github.com/gravitational/teleport
+  - make clean release OS=$OS ARCH=$ARCH
+  environment:
+    ARCH: amd64
+    GOCACHE: /tmp/push-build-darwin-amd64/go/cache
+    GOPATH: /tmp/push-build-darwin-amd64/go
+    OS: darwin
+    WORKSPACE_DIR: /tmp/push-build-darwin-amd64
+- name: Clean up exec runner storage (post)
+  commands:
+  - set -u
+  - chmod -R u+rw $WORKSPACE_DIR
+  - rm -rf $WORKSPACE_DIR/go $WORKSPACE_DIR/.ssh
+  environment:
+    WORKSPACE_DIR: /tmp/push-build-darwin-amd64
+- name: Send Slack notification (exec)
+  commands:
+  - |2
 
-  - name: Build Mac artifacts
-    environment:
-      GOPATH: /tmp/push-build-darwin-amd64/go
-      GOCACHE: /tmp/push-build-darwin-amd64/go/cache
-      OS: darwin
-      ARCH: amd64
-      WORKSPACE_DIR: /tmp/push-build-darwin-amd64
-    commands:
-      - set -u
-      - cd $WORKSPACE_DIR/go/src/github.com/gravitational/teleport
-      - make clean release OS=$OS ARCH=$ARCH
+    export DRONE_BUILD_LINK="${DRONE_SYSTEM_PROTO}://${DRONE_SYSTEM_HOSTNAME}/${DRONE_REPO_OWNER}/${DRONE_REPO_NAME}/${DRONE_BUILD_NUMBER}"
+    export GOOS=$(go env GOOS)
+    export GOARCH=$(go env GOARCH)
+  - |2-
 
-  - name: Clean up exec runner storage (post)
-    environment:
-      WORKSPACE_DIR: /tmp/push-build-darwin-amd64
-    commands:
-      - set -u
-      - chmod -R u+rw $WORKSPACE_DIR
-      - rm -rf $WORKSPACE_DIR/go $WORKSPACE_DIR/.ssh
+    curl -sL -X POST -H 'Content-type: application/json' --data "{\"text\":\"Warning: \`$GOOS-$GOARCH\` artifact build failed for [\`${DRONE_REPO_NAME}\`] - please investigate immediately!\nBranch: \`${DRONE_BRANCH}\`\nCommit: \`${DRONE_COMMIT_SHA}\`\nLink: $DRONE_BUILD_LINK\"}" $SLACK_WEBHOOK
```

Comment:
Either:
- change the environment variable name in the `environment` section to `SLACK_WEBHOOK`
- change the inline reference here to `SLACK_WEBHOOK_DEV_TELEPORT`

...otherwise the Slack stuff will fail to work due to a missing URL.

Reply:
Good catch, fixed.